### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Authentication_Screen Reader: Narrator is not reading the error message when we  turning off the windows authentication button

### ISSUE THREAD

**Issue by InduPriya1805** [Microsoft/IIS.WebManager#116]
GitHubTags: #A11y_Manage.IIS.Net; #A11yMAS; #A11yTCS; #C+E-H; #DesktopWeb; #Win10; #.NETWebsites; #Edge; #ScreenReader; #Narrator; #MAS1.3.1

**Environment Details:**
URL: https://manage.iis.net/webserver/authentication
Browser Details: Microsoft Edge 42.17134.1.0
Screen Reader: Narrator
MAS ID Violated: MAS 1.3.1

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till Web Server and press enter
5. Tab till authentication and press enter
6. Tab till windows tab and press enter
7. Turn off the windows authentication button and verify whether narrator is reading the error message displayed

**Actual Result:**
Narrator is not reading the error message when we  are turning off the windows authentication button, it is reading just as group

**Expected Result:**
Narrator should read the error message displayed when we are turning off the windows authentication button

**Note:**
This issue has been observed in all the pages where we are getting the error message

**User Impact:**
Users with disabilities finds its difficult in working through the application if narrator doesnot read the error message

**Attachments:**
[Bug116_web server_authentication_narrator not reading error message.pptx](https://github.com/Microsoft/IIS.WebManager/files/2326743/Bug116_web.server_authentication_narrator.not.reading.error.message.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
This issue will be fixed when we use WAC SDK controls.

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
